lib/health.js

```diff
@@ -170,6 +170,38 @@ const cmdExec = (cmd, callback) => {
   });
 };
 
+const getSmartStatusOfDrive = (driveName, callback) => {
+  const cmd = `smartctl -H /dev/${driveName}`;
+  api.cp.exec(cmd, (error, stdout) => {
+    if (error) callback(error);
```

Comment:
You don't return after calling the callback here, thus it is possible that the callback will be called twice here.
Also, I think it'd be better to not ignore `stderr`, and print it in this branch/pass it to the callback.